app/views/pages/index.html.erb

```diff
@@ -1,14 +1,55 @@
-<div class="container text-center">
-  <h1>Pages#index</h1>
-  <p>Find me in app/views/pages/index.html.erb</p>
+<div class="container">
   <% if current_user %>
-    <h2>Logged-in User:</h2>
+    <div class="row">
+      <div class="col col-lg-4">
+        <%= link_to 'Create Bet', new_bet_path %>
+      </div>
+      <div class="col col-lg-4">
+      </div>
+      <div class="col col-lg-3">
+      </div>
```

Comment:
Up to you for style preferences but I think we could single line these last couple lines
`<div class="col col-lg-3></div>` etc.